flink-connectors/flink-connector-elasticsearch-base/src/main/java/org/apache/flink/connector/elasticsearch/table/IndexGeneratorFactory.java

```diff
@@ -240,6 +273,11 @@ boolean checkIsDynamicIndexWithFormat(String index) {
             return dynamicIndexTimeExtractPattern.matcher(index).matches();
         }
 
+        /** Check generate a dynamic index is based on processing time. */
+        boolean checkIsDynamicIndexWithProcTimeFormat(String index) {
```

Comment:
```suggestion
        boolean checkIsDynamicIndexFromSystemTimeFormat(String index) {
```

Reply:
Hi, @hackergin @alpreu @matriv , Thanks for your nice work.
I want to talk more about the diff between `proctime()` and `now()`, they have different return type as well as the meaning in Flink SQL, the first one is `TIMESTAMP_LTZ *PROCTIME*`， the later one is `TIMESTAMP_LTZ(3)`, I think the PR description is improper with our current implementation. I suggest changing the issue/PR description to that we supports a system time function(`now()` and `current_timestamp`)  in index pattern, this should be more clear and won't mislead users. WDYT?

Reply:
@leonardBang  I agree with you , and I'll change the describetion  of this issue/PR later.